tests/integration/kubernetes/k8s-job.bats

```diff
@@ -9,7 +9,9 @@ load "${BATS_TEST_DIRNAME}/../../common.bash"
 load "${BATS_TEST_DIRNAME}/tests_common.sh"
 
 setup() {
-	[ "${KATA_HYPERVISOR}" = "qemu-coco-dev" ] || skip "Test not stable on qemu-coco-dev. See issue #10616"
+	if [ "${KATA_HYPERVISOR}" == "qemu-coco-dev" ]; then
+		"Test not stable on qemu-coco-dev. See issue #10616"
```

Comment:
@stevenhorsman , are we missing a "skip " at the beginning of this line? 

I thought "skip message" was the proper way to skip a test, as other tests do - e.g., https://github.com/kata-containers/kata-containers/blob/79495379e2365a5136268ca8b88a91f21fed00f7/tests/integration/kubernetes/k8s-block-volume.bats#L13

Reply:
We absolutely are. Sorry!